openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/infinite-scroll/infinite-scroll.component.ts

```diff
@@ -85,6 +92,10 @@ export class InfiniteScrollComponent<T> implements OnInit, OnChanges, OnDestroy
 
   }
 
+  isHorizontalOrientation(): boolean {
+    return this.orientation && this.orientation === 'horizontal';
```

Comment:
If this.orientation is undefined then it would not equal 'horizontal'
```suggestion
    return this.orientation === 'horizontal';
```

Reply:
touché